.github/workflows/spack_ci_bridge.yml

```
@@ -1,6 +1,10 @@
 name: spack_ci_bridge
 
-on: [push, pull_request]
+on:
+  pull_request:
+    branches-ignore:
```

I think this should be
```suggestion
    tags-ignore:
```

I see a bunch of `flux-write-check-**` tags in https://github.com/spack/spack-infrastructure/tags, and https://fluxcd.io/legacy/flux/faq/#why-does-my-ci-pipeline-keep-getting-triggered mentions `flux-sync` is also a tag, not a branch.